src/components/results-list/ResultsDataFetcher.tsx

```diff
@@ -0,0 +1,136 @@
+import { FC, useEffect, useState, useContext } from "react";
+import { FhirContext } from "../fhir/FhirContext";
+
+import LoadingSpinner from "../UI/LoadingSpinner";
+import ModalWrapper from "../UI/ModalWrapper";
+import { ModalState } from "../UI/ModalWrapper";
+import ResultsList from "../results-list/ResultsList";
+
+import { Patient, Observation } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/models-r4";
+
+const FHIR_URL = process.env.REACT_APP_FHIR_URL;
+
+type trimmedObservation = { cDnaChange: string | undefined; observationId: string | undefined };
+type patientResult = {
+  firstName: string | undefined;
+  lastName: string | undefined;
+  patientId: string | undefined;
+  observations: trimmedObservation[];
+};
+export type parsedResultsModel = patientResult[];
+
+const ResultsDataFetcher: FC = () => {
+  const ctx = useContext(FhirContext);
+  const [modal, setModal] = useState<ModalState | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [parsedResults, setParsedResults] = useState<parsedResultsModel | null>(null);
+
+  useEffect(() => {
+    const observationQueryUrl = `${FHIR_URL}/Observation?_profile=http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant&_include=Observation:subject`;
```

Comment:
We don't need the FHIR URL in this, and can remove the import
```suggestion
    const observationQueryUrl = `/Observation?_profile=http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant&_include=Observation:subject`;
```